packages/kyt-starter-static/src/routes/index.js

```diff
@@ -4,18 +4,18 @@ import Route from 'react-router/lib/Route';
 import IndexRoute from 'react-router/lib/IndexRoute';
 import App from '../components/App';
 
-// Webpack 2 supports ES2015 `System.import` by auto-
+// Webpack 2 supports ES2015 `import()` by auto-
 // chunking assets. Check out the following for more:
-// https://gist.github.com/sokra/27b24881210b56bbaff7#code-splitting-with-es6
+// https://webpack.js.org/guides/migrating/#code-splitting-with-es2015
 
 const importHome = (nextState, cb) => {
-  System.import('../components/Home')
+  import('../components/Home')
```

Comment:
Seems like these `imports` aren't getting correctly parsed by eslint, since I'm seeing errors like `error  Parsing error: 'import' and 'export' may only appear at the top level`. Not sure if maybe babel-eslint would be the solution?